src/main/java/taxi/web/filters/AuthenticationFilter.java

```diff
@@ -35,7 +35,7 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         }
 
         Long driverId = (Long) req.getSession().getAttribute(DRIVER_ID);
-        if (driverId == null || driverService.get(driverId) == null) {
+        if (req.getSession().getAttribute(DRIVER_ID) == null) {
```

Comment:
and another little moment: what will happen if logged in user deletes themselves?